bims/models/biological_collection_cluster.py

```diff
@@ -0,0 +1,27 @@
+# coding=utf-8
+"""Boundary model definition.
+"""
+
+from django.contrib.gis.db import models
+from bims.models.boundary import Boundary
+
+
+class BiologicalCollectionCluster(models.Model):
+    """Biological Cluster model."""
+    boundary = models.ForeignKey(
+        Boundary,
+        on_delete=models.CASCADE
+    )
+    module = models.CharField(
+        max_length=128
+    )
+    site_count = models.IntegerField(default=0)
+    survey_count = models.IntegerField(default=0)
```

Comment:
@cchristelis even though the client did not ask for record and survey level data I think we should store it for clusters anyway - it can give us interesting stats internally.